src/tiles/selection-tile.component.ts

```diff
@@ -88,6 +88,7 @@ export class SelectionTile {
 	keyboardInput(event) {
 		if (event.key === "Enter" || event.key === "Spacebar" || event.key === " ") {
 			this.selected = !this.selected;
+			this.input.nativeElement.dispatchEvent(new Event("change"));
```

Comment:
What's the benefit of doing this over calling `this.change.emit`?

Reply:
Good point, it accomplishes the same thing in the end.